kudu/pom.xml

```diff
@@ -0,0 +1,299 @@
+            </configuration>
+            <goals>
+              <goal>antlr4</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- The reason we are duplicating the plugin from parent is to take care of checkstyle violations in Antlr4 autogenerated code -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <!-- <maxAllowedViolations>2709</maxAllowedViolations> -->
+          <logViolationsToConsole>${checkstyle.console}</logViolationsToConsole>
+          <excludes>**/KuduSQLExpression*</excludes> <!-- To account for violations from the Antlr autogenerated code -->
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <configuration>
+          <excludes combine.children="append">
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpression.tokens</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionLexer.tokens</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionLexer.java</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionListener.java</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionParser.java</exclude>
+            <exclude>src/main/antlr4/KuduSQLExpression.g4</exclude>
+          </excludes>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>2.4</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test-jar</goal>
+            </goals>
+            <phase>package</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- create resource directory for xml javadoc-->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>createJavadocDirectory</id>
+            <phase>generate-resources</phase>
+            <configuration>
+              <tasks>
+                <delete dir="${project.build.directory}/generated-resources/xml-javadoc"/>
+                <mkdir dir="${project.build.directory}/generated-resources/xml-javadoc"/>
+              </tasks>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- generate javdoc -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
get rid of this and related configuration, see previous comment

Reply:
Removed